components/brave_wallet/browser/zcash/rust/zcash_encoding/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
```

Comment:
Cargo.toml shouldn't be necessary here and seems problematic because it would have to be manually kept in sync with BUILD.gn